package.json

```diff
@@ -17,6 +17,25 @@
     "storybook:serve-static": "lerna run storybook:serve-static --stream",
     "storybook:deploy": "lerna run storybook:deploy",
     "test": "cross-env-shell \"lerna run $LERNA_ARGS test --stream\"",
+    "ci:test:tools": "cross-env-shell \"lerna run $LERNA_ARGS ci:test:tools --stream\"",
+    "ci:test:services": "cross-env-shell \"lerna run $LERNA_ARGS ci:test:services --stream\"",
+    "ci:test:atoms": "cross-env-shell \"lerna run $LERNA_ARGS ci:test:atoms --stream\"",
+    "ci:test:molecules": "cross-env-shell \"lerna run $LERNA_ARGS ci:test:molecules --stream\"",
+    "ci:test:organisms": "cross-env-shell \"lerna run $LERNA_ARGS ci:test:organisms --stream\"",
+    "ci:test:pages": "cross-env-shell \"lerna run $LERNA_ARGS ci:test:pages --stream\"",
+    "ci:test:templates": "cross-env-shell \"lerna run $LERNA_ARGS ci:test:templates --stream\"",
```

Comment:
Not an expert, but can't we have one command which expects a parameter so we don't have so much duplication? So then we can run something like `yarn ci:test --tools`.

Reply:
I agree this looks dreadful - apparently this is something that is supported in npm, but not easily in yarn. Will look to see if we can do something with an external node file.